backend/curriculum_tracking/management/commands/fix_project_and_topic_complete_times.py

```diff
@@ -0,0 +1,28 @@
+#update the completed time on all projects and topics
+
+from django.core.management.base import BaseCommand
+from curriculum_tracking.models import RecruitProject,AgileCard,RecruitProjectReview
+from curriculum_tracking.constants import REVIEW_STATUS_CHOICES
+
+
+
+
+def fix_complete_times():
+    cards = AgileCard.objects.filter(status=AgileCard.COMPLETE)
```

Comment:
This iterates over ALL the complete cards. You can use filters to get only those cards that dont have complete dates.

```
cards = (
            AgileCard.objects.filter(status=AgileCard.COMPLETE)
              .filter(recruit_project__complete_time__isnull=True)
        )
```